src/sidebar/components/hypothesis-app.js

```diff
@@ -56,23 +59,34 @@ function HypothesisAppController(
   // used by templates to show an intermediate or loading state.
   this.auth = { status: 'unknown' };
 
-  // App dialogs
-  this.helpPanel = { visible: false };
-
   // Check to see if we're in the sidebar, or on a standalone page such as
   // the stream page or an individual annotation page.
-  this.isSidebar = $window.top !== $window;
+  this.isSidebar = isSidebar();
```

Comment:
The controller still needs to have a `isSidebar` attribute because the angular template makes use of it.